notion/block.py

```diff
@@ -647,6 +655,31 @@ def _str_fields(self):
         return super()._str_fields() + ["title", "collection"]
 
 
+class CollectionViewBlockViews(Children):
+
+    child_list_key = "view_ids"
+
+    def _get_block(self, view_id):
+        return self._parent._client.get_collection_view(view_id, collection=self._parent.collection)
```

Comment:
Here and elsewhere: you can just do `self._client.`, as we set `self._client = parent._client` in `Children.__init__`.